package-lock.json

```diff
@@ -2381,9 +2381,9 @@
       }
     },
     "@openmrs/esm-api": {
-      "version": "3.1.10-pre.201",
```

Comment:
yes, the package-lock diff is actually necessary. need the new openmrs-core lib since it has the newest single-spa version